sync/infrahub-sync/infrahub_sync/__init__.py

```diff
@@ -57,6 +107,63 @@ def model_loader(self, model_name: str, model):
 
 
 class DiffSyncModelMixin:
+    @staticmethod
```

Comment:
I think it would be best to use `classmethod` here instead of `staticmethod` to remain compatible with class inheritance and make it easier to change the behavior of the filters & transforms for a specific adapter if needed